examples/pytorch/gcn/README.md

```diff
@@ -2,33 +2,29 @@ Graph Convolutional Networks (GCN)
 ============
 
 - Paper link: [https://arxiv.org/abs/1609.02907](https://arxiv.org/abs/1609.02907)
-- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original code is 
-implemented with Tensorflow for the paper. 
+- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original implementation is based on Tensorflow.
 
-Dependencies
-------------
-- PyTorch 0.4.1+
-- requests
-
-``bash
-pip install torch requests
-``
-
-Codes
+Available examples
 -----
-The folder contains three implementations of GCN:
-- `gcn.py` uses DGL's predefined graph convolution module.
-- `gcn_mp.py` uses user-defined message and reduce functions.
-Modify `train.py` to switch between different implementations.
+The folder includes an implementations of GCN using DGL intrinsic graph convolution module.
```

Comment:
L7-9 also do not seem to be necessary.

Reply:
Okay. I will remove it here. For Graphsage, when we plan to include multiple base implementations (full, mini-batch, link-pred), it might be helpful to add such section.